Jenkinsfile

```diff
@@ -40,7 +40,8 @@ pipeline {
                        queue: 'osci-pipelines-queue-14'
                    ],
                    checks: [
-                       [field: '$.update.release.dist_tag', expectedValue: triggerDistTagRegEx]
+                       [field: '$.update.release.dist_tag', expectedValue: '^(f[3-9]{1}[0-9]{1}|epel10.[0-9]+)$'],
+                       [field: '$.update.release.branch', expectedValue: '^(f[3-9]{1}[0-9]{1}|rawhide|epel10)$']
```

Comment:
Out of curiosity, can you please point me to documentation that explains how the `checks` work? In my earlier attempt, I was going to do it the same way you did here but based on our other projects I got the impression that the `[field: ...]` lines are AND'd rather then OR'd, so I ended up duplicating the whole `rabbitMQSubscriber` section, IIRC.

Reply:
They are indeed AND'd (and this is, afaik, not documented anywhere...). So the current settings would trigger even on updates where the target branch was Rawhide, and the dist_tag was epel. Which would be incorrect, but it would also indicate a problem somewhere earlier in the process.

Your approach was good, but the (potential) problem was the re-use of the same queue. I think there was a possibility that those 2 RabbitMQ providers would be stealing messages from each other, if they were configured to use the same queue. I didn't want to go through the hassle of figuring out the process for creating new queues, so I just modified the trigger not to require multiple queues.